PowerSimData2Switch.py

```diff
@@ -0,0 +1,432 @@
+# -*- coding: utf-8 -*-
+
+"""
+Created on Wed Mar 16 12:27:29 2021
+
+@author: YifanLi @ Breakthrough Energy Science
+"""
+
+# This code transfers generation and transmission data from PowerSimData scenario grid to Switch2.0 input format.
+# Generate data inputs needed for all basic modules of Switch2 expansion optimization model in .csv format.
+# No geographical reduction will be performed, all data maintains original nodal model resolution.
+# Refer to TemporalSlicing.py for time series related input data development.
+# Investment cost data from NREL ATB.
+
+
+import os, sys, csv
+import pandas as pd
+import numpy as np
+import math as m
+
+# user_defined inputs.
+
+base_year = input("Please enter base study year (normally PowerSimData scenario year): ")

```

Comment:
Typically, we enclose all functional code (things besides defining constants) into one or more functions. Otherwise, all of this code will be executed upon import. You can still get the same ability to run from the command line by adding a `if __name__ == "__main__":` block, which will execute things as necessary.